ext/intl/php_intl.stub.php

```diff
@@ -344,7 +344,7 @@ function msgfmt_get_error_message(MessageFormatter $formatter): string {}
 
 /* normalizer */
 
-function normalizer_normalize(string $string, int $form = Normalizer::FORM_C): string|false {}
+function normalizer_normalize(string $string, int $form = Normalizer::FORM_C): string|false|null {}
```

Comment:
I adjusted this one to not have the null type in https://github.com/php/php-src/commit/5dc995df375571489d9149fdccf258c0bd123317.